tests/test_sample_file.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from unittest import mock
+from singer_encodings import json_schema
+from singer_encodings import csv
+import csv as _csv
+
+class SFTPConnection:
```

Comment:
@harshpatel4crest Since `singer-encodings` is a generic framework, please we should not use specific Connection names like 'SFTPConnections'